<?xml version='1.0' encoding='UTF-8' standalone='no'?>
<section xmlns="http://docbook.org/ns/docbook" version="5.0" xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="_classgame_1_1utils_1_1exceptions_1_1PlayerNameAlreadyExists" xml:lang="en-US">
<title>game.utils.exceptions.PlayerNameAlreadyExists Class Reference</title>
<indexterm><primary>game.utils.exceptions.PlayerNameAlreadyExists</primary></indexterm>
<para>Inheritance diagram for game.utils.exceptions.PlayerNameAlreadyExists:    <informalfigure>
        <mediaobject>
            <imageobject>
                <imagedata width="50%" align="center" valign="middle" scalefit="0" fileref="classgame_1_1utils_1_1exceptions_1_1PlayerNameAlreadyExists.png"></imagedata>
            </imageobject>
        </mediaobject>
    </informalfigure>
</para>
<simplesect>
    <title>Public Member Functions    </title>
        <itemizedlist>
            <listitem><para><link linkend="_classgame_1_1utils_1_1exceptions_1_1PlayerNameAlreadyExists_1a33383c76fb3a7f4d2923e110d8d43ae8">__init__</link> (self, message=&quot;Your current player name is already taken, please change it.&quot;)</para>
</listitem>
        </itemizedlist>
</simplesect>
<section>
<title>Detailed Description</title>

<para><literallayout><computeroutput>Class for creating the exception caused by joining a server with an in-game name that is already taken.
</computeroutput></literallayout> </para>
</section>
<section>
<title>Constructor &amp; Destructor Documentation</title>
<anchor xml:id="_classgame_1_1utils_1_1exceptions_1_1PlayerNameAlreadyExists_1a33383c76fb3a7f4d2923e110d8d43ae8"/><section>
    <title>__init__()</title>
<indexterm><primary>__init__</primary><secondary>game.utils.exceptions.PlayerNameAlreadyExists</secondary></indexterm>
<indexterm><primary>game.utils.exceptions.PlayerNameAlreadyExists</primary><secondary>__init__</secondary></indexterm>
<para><computeroutput>game.utils.exceptions.PlayerNameAlreadyExists.__init__ ( self,  message = <computeroutput>&quot;Your&#160;current&#160;player&#160;name&#160;is&#160;already&#160;taken,&#160;please&#160;change&#160;it.&quot;</computeroutput>
)</computeroutput></para></section>
<para>
The documentation for this class was generated from the following file:</para>
<link linkend="_exceptions_8py">exceptions.py</link></section>
</section>
